<launch>
<!-- Launch file for Place recognition example -->

  <arg name="rviz" default="true" />

  <node 
		pkg="btc_desc" 
		type="kitti_btc_place_recognition" 
		name="kitti_btc_place_recognition" 
		output="screen" 
	/>

  <param name="cloud_overlap_thr" type="double" value = "0.5"/>
  <param name="read_bin" type="bool" value="true" />
  <param name="setting_path" type="string" value = "$(find btc_desc)/config/config_outdoor.yaml"/>
  <param name="pcds_dir" type="string" value = "/home/sti/2TB_HDD/Rumble/workspace/robot_slam/BTC/src/my_btc/example/KITTI/data/kitti00/cloud/velodyne"/>
  <param name="pose_file" type="string" value = "/home/sti/2TB_HDD/Rumble/workspace/robot_slam/BTC/src/my_btc/example/KITTI/data/kitti00/pose/tum_format_pose.txt"/>
  <param name="enable_transform" type="bool" value="true" />

  <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find btc_desc)/rviz_cfg/loop.rviz" />
  </group>

</launch>